scripts/link_bb.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+from os import path, makedirs
+from subprocess import run
+
+from qcore import simulation_structure
+from qcore import constants
+from shared_workflow.shared_automated_workflow import add_to_queue
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "source_root_dir", type=str, help="path to previous completed vaildation run"
+    )
+    parser.add_argument(
+        "destination_root_dir", type=str, help="path to the destination root directory"
+    )
+    parser.add_argument(
+        "fault_selection_list", type=str, help="a file that contains list of faults"
+    )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def main(source_root_dir, destination_root_dir, fault_selection_list):
+    faults = {}
+    with open(fault_selection_list, "r") as fault_file:
+        for line in fault_file.readlines():
+            fault, count, *_ = line.split()
+            count = int(count[:-1])
+            faults.update({fault: count})
+    for fault, count in faults.items():
+        for rel_no in range(count):
```

Comment:
range(start, end+1)

Reply:
> I'd like a separate folder for the scripts below rather than putting them in the scripts folder. The scripts folder is way too hard to find what you are looking for.
> 
> Aside, this new functionality should be a separate pull-request rather than making this pull-request bigger and hence take longer in the review stage.

good call. i'll move them to a new PR, as the functions are unrelated to this one.